resourcematch-app/renderer/_default.page.server.tsx

```diff
@@ -0,0 +1,46 @@
+export { render }
+// See https://vike.dev/data-fetching
+export const passToClient = ['pageProps', 'urlPathname']
+
+import ReactDOMServer from 'react-dom/server'
+import { PageShell } from './PageShell'
+import { escapeInject, dangerouslySkipEscape } from 'vike/server'
+import logoUrl from './logo.svg'
```

Comment:
I think this needs to be removed along with reference below (currently builds break).